.github/workflows/test.yml

```
@@ -32,3 +37,4 @@ jobs:
         with:
           start: npm start
           wait-on: http://localhost:3000
+          env: CYPRESS_VALID_EMAIL=${{ secrets.CYPRESS_VALID_EMAIL }},CYPRESS_VALID_PASSWORD=${{ secrets.CYPRESS_VALID_EMAIL }}
```

Can we adjust it like this?
```suggestion
          env: 
            CYPRESS_VALID_EMAIL: ${{ secrets.CYPRESS_VALID_EMAIL }}
            CYPRESS_VALID_PASSWORD: ${{ secrets.CYPRESS_VALID_EMAIL }}
```

---

We cannot. When I adjust followed your suggestion, the CI is failed
https://github.com/gutakk/react-surveyor/runs/2873063227?check_suite_focus=true

After adjust back, the CI is passed
https://github.com/gutakk/react-surveyor/runs/2873108825?check_suite_focus=true